rasa/nlu/training_data/formats/luis.py

```diff
@@ -18,13 +19,24 @@ def read_from_json(self, js: Dict[Text, Any], **kwargs: Any) -> "TrainingData":
         training_examples = []
         regex_features = []
 
-        # Simple check to ensure we support this luis data schema version
-        if not js["luis_schema_version"].startswith("2"):
+        # Simple check to ensure we support this luis data schema
+        if not (
+            "regex_features" in js
+            and "utterances" in js
+            and "text" in js["utterances"][0]
+            and "intent" in js["utterances"][0]
+            and "entities" in js["utterances"][0]
```

Comment:
Why these data is enough to validate if a `json` file is LUIS compatible?
Appears to be generic to me.
Is there any **specific** information that "only" LUIS json files have? Because the previous code verifies **specific** information inside the json file.

Reply:
This data is being used by RASA to train the NLU and until version 5 this is the format that LUIS json file use. So a LUIS json will only be incompatible when one or more of this informations are changed or not present.